packages/shared/src/components/cards/SharedPostCardFooter.tsx

```diff
@@ -20,26 +20,29 @@ export const SharedPostCardFooter = ({
     <div
```

Comment:
Should all the changes in this file be behind the experiment? 
If yes - I think we should copy paste this file in the v1 folder within cards which contains all the files that get changed conditionally by the feed layout experiment.

Reply:
great catch, my bad, yeah, this should be copied under V1 👍